src/SceneKit/Defs.cs

```diff
@@ -30,7 +30,11 @@ public enum SCNGeometryPrimitiveType : nint {
 		Triangles,
 		TriangleStrip,
 		Line,
-		Point
+		Point,
+		[Introduced (PlatformName.iOS, 10, 0)]
+		[Introduced (PlatformName.TvOS, 10, 0)]
+		[Introduced (PlatformName.MacOSX, 10, 12)]
```

Comment:
it's fine but a bit verbose, please use `[TV (10, 0), Mac (10, 12), iOS (10, 0)]` on a single line to ease readability